docs/api/Components.md

```diff
@@ -182,31 +157,29 @@ describe('TestComponent', function () {
             // React must be required after window is set
             React = require('react');
```

Comment:
I'd like to also consider [`lab`](https://github.com/hapijs/lab) as another option. JSX transformation is possible.